landoapi/auth.py

```diff
@@ -29,6 +29,10 @@
 
 logger = logging.getLogger(__name__)
 
+MOZILLIANS_PREFIX = "mozilliansorg_"
+TREESTATUS_USERS = f"{MOZILLIANS_PREFIX}treestatus_users"
+TREESTATUS_ADMIN = f"{MOZILLIANS_PREFIX}treestatus_admins"
```

Comment:
```suggestion
TREESTATUS_PREFIX = "mozilliansorg_treestatus_"
TREESTATUS_USERS = f"{TREESTATUS_PREFIX}users"
TREESTATUS_ADMIN = f"{TREESTATUS_PREFIX}admins"
```

Reply:
The group names in Mozillians are `treestatus_users` and `treestatus_admins`, and the prefix is an implementation detail of how the groups are returned from SSO. I'd like to keep the `treestatus_` part outside of the Mozillians prefix since including it there isn't an accurate reflection of how they're represented in SSO.

I could make a variable like `USERS_GROUP = "treestatus_users"` and then do `USER_MOZILLIANS_GROUP = f"{MOZILLIANS_PREFIX}{USERS_GROUP}"` if that makes things clearer, or maybe just add a comment.